changelog.d/9636.bugfix

```diff
@@ -0,0 +1 @@
+Checks if passwords are allowed before setting it for the user.
```

Comment:
I wonder if we should consolidate the various three (?) PRs we've made around this into a single changelog entry. What do you think?

Reply:
Yes, we can do it. But the first one (#9587) is already in 1.30.0rc1 included.

Reply:
Ah, well maybe just for the last two then!